civihr_employee_portal/views/templates/_views-view--generic-display-output.tpl.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * @file
+ * Main view template.
+ *
+ * Variables available:
+ * - @var array $classes_array: An array of classes determined in
+ *   template_preprocess_views_view(). Default classes are:
+ *     .view
+ *     .view-[css_name]
+ *     .view-id-[view_name]
+ *     .view-display-id-[display_name]
+ *     .view-dom-id-[dom_id]
+ * - @var string $classes: A string version of $classes_array for use in the class attribute
+ * - @var string $css_name: A css-safe version of the view name.
+ * - @var string $css_class: The user-specified classes names, if any
+ * - @var string $header: The view header
+ * - @var string $footer: The view footer
+ * - @var string $rows: The results of the view query, if any
+ * - @var string $empty: The empty text to display if the view is empty
+ * - @var string $pager: The pager next/prev links to display, if any
+ * - @var string $exposed: Exposed widget form/info to display
+ * - @var string $feed_icon: Feed icon to display, if any
+ * - @var string $more: A link to view more, if any
+ *
+ * @ingroup views_templates
+ */
+  $display_options = $view->display['default']->display_options;
```

Comment:
@powdevel avoid any processing in tpl files please use preprocess functions to handle.

Reply:
@hitesh-jain Ok moving this logic to a preprocess function.